GeneralsMD/Code/Libraries/Source/debug/debug_except.cpp

```diff
@@ -173,12 +173,18 @@ void DebugExceptionhandler::LogFPURegisters(Debug &dbg, struct _EXCEPTION_POINTE
     double fpVal;
 
     // convert from temporary real (10 byte) to double
+#ifdef _MSC_VER
     _asm
     {
       mov eax,value
       fld tbyte ptr [eax]
       fstp qword ptr [fpVal]
     }
+#else
+    __float80 fp80val;
```

Comment:
This assumes a lot IMO, should be gated to GCC and X86 only as I'm not sure the type exists elsewhere and give an unimplemented error or message otherwise?